### DISCUSSION TITLE
multiple backup location possible

### DISCUSSION THREAD

**Discussion by changemenemo** [standardnotes/forum#2614] [Suggestions]
Hi,
I would like to suggest to have the possibility to configure multiple backup location.
I have different folder with different nextcloud etc and I would like to save it to multiple location.
It can be in a queue for example  or anything a like.
And I don't like background services to actually make copies

**Comment by JaspalSuri** [comment]
Hi, I'll take note of this feature request. Thanks for the feedback!
